test/services/allocation/AssessorAllocationServiceSpec.scala

```diff
@@ -80,14 +84,35 @@ class AssessorAllocationServiceSpec extends BaseServiceSpec {
     }
   }
 
+  "Allocate candidate" must {
```

Comment:
If you decide to move allocate candidates to its own service, you will have to create a separate spec I guess for this test.